app/assets/stylesheets/user.css.scss

```diff
@@ -2,25 +2,27 @@
     padding-top: 150px;
 }
 
+.form-background {
+  display: block;
+  background-color: #eeeeee;
+  padding: 150px 0 20px 20px;
+  text-decoration: none;
+  height: 475px;
+}
+
 #table-title {
-    font-size: 24px;
+    font-size: 32px;
+    font-weight:800;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space